components/constituency_lookup/types.tsx

```diff
@@ -12,13 +12,19 @@ type Address = {
   slug: string;
 };
 
-type ErrorCode = "POSTCODE_INVALID" | "POSTCODE_NOT_FOUND" | "SERVER_ERROR";
+type PostCodeErrorCode =
```

Comment:
Maybe `ServerPostcodeErrorCode` is more descriptive of the intent here?
Then the "all" one can just be `PostcodeErrorCode`? :thinking: